mods/ts/chrome/ingame-observer.yaml

```diff
@@ -0,0 +1,160 @@
+Container@OBSERVER_WIDGETS:
```

Comment:
I think it would make sense for *this* version (shared by d2k and ts) to be in `mods/common/chrome/`, and then let RA define its own.